packages/react-native-layout/src/components/container/Container.props.ts

```diff
@@ -0,0 +1,11 @@
+import type { LayoutChangeEvent } from 'react-native';
```

Comment:
not a big fan of putting these props in a separate file. seems like a lot of overhead for 10 lines of code